src/main/java/com/karankumar/bookproject/ui/shelf/BookFilters.java

```diff
@@ -0,0 +1,44 @@
+package com.karankumar.bookproject.ui.shelf;
+
+import com.karankumar.bookproject.backend.entity.Book;
+
+class BookFilters {
+    private String bookTitle;
+    private String bookAuthor;
+
+    private String getBookTitle() {
+        return bookTitle.toLowerCase();
+    }
```

Comment:
I believe it will be less readable if  we inline it. I think it's more import to have a readable code rather than smaller code base :slightly_smiling_face:  But sure let's do it :slightly_smiling_face:

Reply:
As a compromise, what about changing the method name to something like `getCaseInsensitiveBookTitle()`?

Reply:
I'm glad that you tried to meet halfway :slightly_smiling_face:  But I changed my mind and decided to go with your way :slightly_smiling_face: